controllers/news.controller.js

```diff
@@ -0,0 +1,19 @@
+const { fetchTopics, fetchArticle } = require("../models/news.model");
+const checkIDexists = require("../models/checkIDexists");
+
+const getTopics = (req, res) => {
+  fetchTopics().then((topics) => {
+    res.send(topics);
+  });
+};
+
+const getArticle = (req, res, next) => {
```

Comment:
Please split your models and controllers out to reflect the data they're dealing with (e.g topics-controllers and articles-controllers)